### PR TITLE
Vaccine logistic - API Update Integrated Status from WMS Poslog

### PR DESCRIPTION
**Pull request by rindibudiaramdhan**
## Overview
- Update status Vaccine Request from WMS Poslog. And send email to applicant in each status with specific information.
- Status:
  - BOOKED (booked)
  - DO (do)
  - INTRANSIT (intransit)

## Evidence
- title: Vaccine logistic - API Update Integrated Status from WMS Poslog
- project: Pikobar Logistik
- participants: @rindibudiaramdhan @ayocodingit

### CODE REVIEW

File: app/Models/Vaccine/VaccineRequest.php [hunk 1]
```diff
@@ -18,32 +18,23 @@ class VaccineRequest extends Model
...
-        'letter_number',
-        'letter_file_url',
-        'applicant_file_url',
-        'is_letter_file_final',
-        'is_completed',
-        'is_urgency',
-        'status',
-        'note',
+        'agency_city_id', 'agency_district_id', 'agency_village_id', 'agency_address',
+        'applicant_fullname', 'applicant_position',
```
Comment (ayocodingit) [inline app/Models/Vaccine/VaccineRequest.php:24]: menghindari max 25 baris yah kang hehe

File: app/Mail/Vaccine/VerifiedEmailNotification.php [hunk 2]
```diff
@@ -138,22 +132,50 @@ public function setMessageFinalized()
...
+        $this->texts[] = '';
+        $this->texts[] = '<b>Tahapan Selanjutnya</b>';
+        $this->texts[] = 'Permohonan Logistik Vaksin Anda sedang dikemas untuk dimasukkan ke dalam kendaraan. Selanjutnya, akan dilakukan pengiriman barang sesuai tanggal rencana kirim.';
+        $this->texts[] = '';
+        $this->texts[] = '<b>Tanggal Rencana Kirim</b>';
+        $this->texts[] = 'Barang akan dikirim pada tanggal <b><i>' . date('d-m-Y', strtotime($this->vaccineRequest->delivery_plan_date)) . '</i></b>. Mohon bersiap pada tanggal tersebut.';
+        $this->texts[] = '';
+        $this->setTableValue();
+        $this->setNote();
+    }
```
Comment (ayocodingit) [inline app/Mail/Vaccine/VerifiedEmailNotification.php:151]: Similar blocks of code found in 4 locations. Consider refactoring

ada warning kang rindi

Comment (rindibudiaramdhan) [reply app/Mail/Vaccine/VerifiedEmailNotification.php:151]: iya. ini beda sebenarnya. di next clean code saya coba perbaiki ya

File: app/Models/Vaccine/VaccineRequest.php (hunk 1 above)
Comment (rindibudiaramdhan) [reply app/Models/Vaccine/VaccineRequest.php:24]: iya nih. udah kepenuhan ternyata modelnya.

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**